docs/_articles/en/index.md

```diff
@@ -23,12 +23,12 @@ Code Builder is a browser-based version of the desktop experience, with everythi
 
 ## Additional Resources
 
-- Trailhead: [Build Apps Together with Package Development](https://trailhead.salesforce.com/trails/sfdx_get_started)
+Trailhead:
+- [Build Apps Together with Package Development](https://trailhead.salesforce.com/trails/sfdx_get_started)
 - [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup)
 - [Salesforce DX Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev)
 
-## Videos
-
+Videos:
```

Comment:
It seems like we have a lot of videos here from DF/TDX; could we add the most recent ones here as an archive?

Reply:
I am adding a couple. Please let me know what else you'd like to see. We might have to take these down eventually, but worth adding for now, I guess.